src/relay/transforms/pattern_util.h

```diff
@@ -554,7 +554,10 @@ static inline Expr Reshape(Expr data, Array<Integer> newshape) {
   attrs->newshape = std::move(newshape);
   attrs->reverse = false;
   static const Op& op = Op::Get("reshape");
-  return Call(op, {data}, Attrs(attrs), {});
+  std::vector<int> value{1};
+  auto dummy_newshape_tensor = MakeConstantTensor(DataType::Int(32), {1}, value);
```

Comment:
I think we should change the Reshape API here as well to make it consistent. After that we can change the type of newshape attr to be Expr as well.

Reply:
Anther way is that we can overload Reshape in pattern_util.h to allow newshape to be Array, but convert to a Constant inside it before calling actual Reshape API.